alvr/xtask/src/main.rs

```diff
@@ -255,7 +256,14 @@ fn main() {
                 "check-format" => format::check_format(),
                 "clean" => clean(),
                 "bump" => version::bump_version(version, is_nightly),
-                "clippy" => clippy(),
+                "clippy" => {
+                    // Slightly stupid api but what gives
```

Comment:
I mean ok, but we don't need to put this comment. Maybe let's rename clippy() to clippy_pedantic()?